src/Disruptor/RingBuffer.cs

```diff
@@ -46,13 +47,13 @@ protected RingBuffer(ISequencer sequencer, Type eventType, int bufferPad)
             {
                 throw new ArgumentException("bufferSize must not be less than 1");
             }
-            if (!_bufferSize.IsPowerOf2())
+            if (!((int)_bufferSize).IsPowerOf2())
```

Comment:
I do not understand this change. `_bufferSize` seems to be of type `int`.

Reply:
Probably left after I experimented with making it a `long`